.github/workflows/backport-reaper.yaml

```diff
@@ -0,0 +1,22 @@
+name: Backport branch reaper
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - 'release/*.*'
+
+jobs:
+  remove_backport_branch:
+    if: startsWith(github.event.pull_request.head.ref, 'backport/') && github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete PR head branches
+        # language=bash
+        run: |
+          # safety switch that we only remove backport branches
+          BP=$(echo ${{ github.event.pull_request.head.ref }} | sed 's|backport/||' ) 
```

Comment:
Is this not already covered by the `branches` filter under `on`?

Reply:
It's a double check and I'd like to keep it here. Just to make sure it never ends up deleting main or a release branch if triggered on those branches or something else by mistake/bug

Reply:
Yeah I figured as much, fine by me :+1: